cfme/tests/containers/test_manageiq_ansible_custom_attributes.py

```diff
@@ -36,16 +36,17 @@ def ansible_custom_attributes():
     remove_tmp_files()
 
 
-def verify_custom_attributes(provider, custom_attributes_to_verify):
+def verify_custom_attributes(appliance, provider, custom_attributes_to_verify):
     view = navigate_to(provider, 'Details')
+    appliance.server.browser.refresh()
```

Comment:
Q:
after navigating to `Detail` page you need browser refresh for checking `custom Attributes`. Is there any BZ?

Reply:
So this isn't so much a CloudForms problem more of a nuance with the test. This test adds and removes custom attributes via an ansible playbook. After the first navigation, the view will stay on the details page since it is already there,which causes the test to fail because new custom attributes have been added or removed via ansible. I thought about adding a `ressetter` to the Details class but thought this made more sense.

Reply:
I would actually lean toward using resetter here - any reason not to? Was it still sometimes refreshing before the information was available?

Reply:
Merging for test fixes, this can be considered and easily added to the provider nav.